src/07-mocking-lib-api/index.test.ts

```diff
@@ -1,17 +1,57 @@
-// Uncomment the code below and write your tests
-/* import axios from 'axios';
-import { throttledGetDataFromApi } from './index'; */
+import axios from 'axios';
+import { throttledGetDataFromApi, THROTTLE_TIME } from './index';
+import { AxiosInstance } from 'axios';
+
+jest.mock('axios');
 
 describe('throttledGetDataFromApi', () => {
+  const axiosMock = jest.mocked(axios);
+  const getSpy = jest.fn();
+  const data = 'hello';
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  beforeEach(() => {
+    jest.advanceTimersByTime(THROTTLE_TIME);
```

Comment:
it's not mistake but better to use
```
jest.mock('lodash', () => ({
  throttle: jest.fn((callback) => callback),
}));
```
because you don't need to know how the throttle works inside